### PR TITLE
add buttons to trading header

### LINKED ISSUE #3969: last price not showing right on scalars
Screen Shot 2019-10-01 at 2 49 10 PM

### PR DESCRIPTION
**Pull request by phoebemirman**
part of https://github.com/AugurProject/augur/issues/3733
closes https://github.com/AugurProject/augur/issues/3969

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💫

 This link will always point to this PR's latest changes: https://peek.run/0elhpo9

File: packages/augur-sdk/src/state/getter/Users.ts [hunk 1]
```diff
@@ -532,17 +536,20 @@ export class Users {
     const universe = params.universe
       ? params.universe
       : await augur.getMarket(params.marketId).getUniverse_();
-    const profitLossSummary = await Users.getProfitLossSummary(augur, db, {
+    profitLossSummary = await Users.getProfitLossSummary(augur, db, {
       universe,
       account: params.account,
     });
-
+  } catch(e) {
```
Comment (bthaile) [inline packages/augur-sdk/src/state/getter/Users.ts:543]: leaving try catch in b/c exceptions get swallowed up by sdk

File: packages/augur-ui/src/modules/market-cards/common.tsx [hunk 2]
```diff
@@ -260,7 +247,7 @@ export const OutcomeGroup = (props: OutcomeGroupProps) => {
             min={props.min}
             max={props.max}
             lastPrice={
-              outcomesShow[0].price ? outcomesShow[0].lastPricePercent : null
+              outcomesShow[YES_NO_YES_ID].price ? formatNumber(outcomesShow[YES_NO_YES_ID].price) : null
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/common.tsx]: better to use scalar constants. 

export const SCALAR_DOWN_ID = 1;
export const SCALAR_UP_ID = 2;

File: packages/augur-ui/src/modules/market/components/market-scalar-outcome-display/market-scalar-outcome-display.tsx [hunk 3]
```diff
@@ -3,33 +3,40 @@ import PropTypes from 'prop-types';
 
 import getValue from 'utils/get-value';
 import CustomPropTypes from 'utils/custom-prop-types';
-import { createBigNumber } from 'utils/create-big-number';
+import { createBigNumber, BigNumber } from 'utils/create-big-number';
 import { DashlineLong } from 'modules/common/labels';
 import MarketOutcomeTradingIndicator from 'modules/market/containers/market-outcome-trading-indicator';
 import Styles from 'modules/market/components/market-scalar-outcome-display/market-scalar-outcome-display.styles.less';
+import { YES_NO_YES_ID } from 'modules/common/constants';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/market-scalar-outcome-display/market-scalar-outcome-display.tsx]: use scalar constant for clarity and readability
export const SCALAR_UP_ID = 2;

File: packages/augur-ui/src/modules/market/components/market-scalar-outcome-display/market-scalar-outcome-display.tsx [hunk 4]
```diff
@@ -3,33 +3,40 @@ import PropTypes from 'prop-types';
...
-    const pricePercentage = createBigNumber(percentage || 0)
-      .minus(min)
-      .dividedBy(range)
-      .times(createBigNumber(100)).toNumber();
-
-    return outcome && outcome.price === null
-      ? 50
-      : pricePercentage
-  };
+  const lastPrice = getValue(outcomes[YES_NO_YES_ID], 'price');
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/market-scalar-outcome-display/market-scalar-outcome-display.tsx]: use constant
export const SCALAR_UP_ID = 2;

File: packages/augur-ui/src/modules/market/containers/market-header-reporting.ts [hunk 5]
```diff
@@ -8,24 +8,24 @@ import { updateModal } from 'modules/modal/actions/update-modal';
 import {
   MODAL_CLAIM_MARKETS_PROCEEDS,
   DESIGNATED_REPORTER_SELF,
+  MODAL_REPORTING,
 } from 'modules/common/constants';
 import { NodeStyleCallback } from 'modules/types';
 import { createBigNumber } from '../../../utils/create-big-number';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/containers/market-header-reporting.ts]: might as well fix this reference to create-big-number to use 'modules/....' reference

File: packages/augur-ui/src/modules/market/containers/market-header.ts [hunk 6]
```diff
@@ -1,33 +1,61 @@
...
 
 const mapStateToProps = (state, ownProps) => {
   const market = ownProps.market || selectMarket(ownProps.marketId);
 
+  const { reportingState, consensusFormatted: consensus } = market;
+  let reportingBarShowing = false;
+  const isDesignatedReporter =
+    market.designatedReporter === state.loginAccount.address;
+
+  const hasConsensus =
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/containers/market-header.ts]: can just check (!consensus) consensus is null or it's populated

**Review by bthaile — CHANGES_REQUESTED** [review]
request some nit pic changes

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**